app/src/main/java/com/macbitsgoa/events/home/HomeActivity.java

```diff
@@ -70,4 +82,44 @@ private void populateFeatures() {
         }
         areFeaturesPopulated = true;
     }
+
+
+    @Override
+    public void onBackPressed() {
+        DrawerLayout drawerLayout = findViewById(R.id.drawer_layout);
+
+        if (drawerLayout.isDrawerOpen(GravityCompat.START)) {
+            drawerLayout.closeDrawer(GravityCompat.START);
+        } else {
+            super.onBackPressed();
+        }
+    }
+
+
+    @Override
+    public boolean onNavigationItemSelected(MenuItem menuItem) {
+
+        switch (menuItem.getItemId()) {
+
+            case (R.id.nav_share_app): {
+                Intent shareIntent = new Intent();
+                shareIntent.setAction(Intent.ACTION_SEND);
+                shareIntent.putExtra(Intent.EXTRA_TEXT, "Download this app from PlayStore");
```

Comment:
- use string resource
- add play store link ( see https://developer.android.com/distribute/marketing-tools/linking-to-google-play#OpeningDetails  ans https://developer.android.com/reference/android/content/Context.html#getPackageName() )